### PR TITLE
data viz decluttering/encoding/filters sections

### PR DESCRIPTION
**Pull request by isobelthompson**

### CODE REVIEW

File: src/data-visualisation/decluttering/decluttering.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: collection
+title: "Decluttering and pulling focus"
+description: "Guidance on the importance of decluttering and pulling focus for data visualisation"
+status: DRAFT
+tags: data
+order: 1
+collection_tag: dv-declutter
```
Comment (WORTI3) [inline src/data-visualisation/decluttering/decluttering.md:8]: Remove collection_tag & pagination settings as this is a single article not a collection of articles.

File: src/data-visualisation/decluttering/decluttering.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+layout: collection
```
Comment (WORTI3) [inline src/data-visualisation/decluttering/decluttering.md:2]: Change this to article as this is not a collection

File: src/data-visualisation/decluttering/decluttering.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+title: "Decluttering and pulling focus"
+description: "Guidance on the importance of decluttering and pulling focus for data visualisation"
+status: DRAFT
+tags: data
+order: 1
```
Comment (WORTI3) [inline src/data-visualisation/decluttering/decluttering.md:7]: Order here plays a role. This should be different from the order in: `../accessibility/index.md` unless the order doesn't matter here then it should be removed. In theory it should be higher than the 0 in accessibility if you want the data vis articles to have a particular order.

File: src/data-visualisation/decluttering/decluttering.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+collection_tag: dv-declutter
+pagination:
+  data: collections.dv-declutter
+  size: 50
+  alias: articles
+---
+## Importance of decluttering  
+  
+To ensure users can quickly make effective and informed decisions, it is important to give the user all information necessary, without providing too much to distract the user.
```
Comment (WORTI3) [inline src/data-visualisation/decluttering/decluttering.md:16]: Has a data vis peer reviewed the content?

**Review by WORTI3 — CHANGES_REQUESTED** [review]
Just a couple comments 😃

File: src/data-visualisation/decluttering/decluttering.md (hunk 4 above)
Comment (isobelthompson) [reply src/data-visualisation/decluttering/decluttering.md:16]: Helen wrote the content, I copied up and made slight edits, Helen re-reviewed

File: src/data-visualisation/decluttering/decluttering.md (hunk 3 above)
Comment (isobelthompson) [reply src/data-visualisation/decluttering/decluttering.md:7]: Okay will amend. I will need to change the order on accessibility to be higher then as it is not the first page.

**Review by WORTI3 — APPROVED** [review]

**Review by WORTI3 — APPROVED** [review]

**Pull request merged**